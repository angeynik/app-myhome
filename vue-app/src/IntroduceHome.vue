<template>
  <div>
    <nav style="margin-bottom: 20px;">
      <router-link to="/dashboard">Dashboard</router-link> |
      <router-link to="/smart-home">Smart Home</router-link> |
      <router-link to="/manufact-automatation">Manufact Automation</router-link>
    </nav>
    <h1>Добро пожаловать!</h1>
    <!-- <h2 style="width: 80%; align-content: flex-start;">
      Наша компания предлагает решения в сфере управления и автоматизации как для вашего дома, так и для вашего бизнеса.
    </h2>
    <h2 style="width: 80%; align-content: flex-end;">
      Вы можете выбрать один из разделов и узнать больше о типовых решениях или заказать индивидуальную разработку.
    </h2> -->

    <p>Выберите раздел:</p>
    <div class="introduce-place">
      <button @click="goToDashboard" class="introduce-button color_dark dash">
        <span class="button-text">Dashboard</span>
      </button>
      <button @click="goToSmartHome" class="introduce-button color-light home">
        <span class="button-text">Smart Home</span>
      </button>
      <button @click="goToManufactAutomation" class="introduce-button color-light manuft">
        <span class="button-text">Manufact Automation</span>
      </button>
    </div>
  </div>
</template>

<script>
export default {
  name: 'IntroduceHome',
  methods: {
    goToDashboard() {
      this.$router.push('/dashboard');
    },
    goToSmartHome() {
      this.$router.push('/smart-home');
    },
    goToManufactAutomation() {
      this.$router.push('/manufact-automatation');
    },
  },
};
</script>

<style scoped>

</style>