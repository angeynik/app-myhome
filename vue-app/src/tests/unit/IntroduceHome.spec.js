import { mount } from '@vue/test-utils';
import IntroduceHome from '@/IntroduceHome.vue';

describe('IntroduceHome.vue', () => {
  it('matches the snapshot', () => {
    const wrapper = mount(IntroduceHome);
    expect(wrapper.html()).toMatchSnapshot();
  });
});