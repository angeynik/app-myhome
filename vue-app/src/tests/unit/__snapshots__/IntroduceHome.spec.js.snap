// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroduceHome.vue matches the snapshot 1`] = `
"<div>
  <nav class="header-bottom">
    <router-link-stub to="/login"></router-link-stub> | <router-link-stub to="/dashboard"></router-link-stub> | <router-link-stub to="/smart-home"></router-link-stub> | <router-link-stub to="/manufact-automatation"></router-link-stub> |
    <!-- Profile (доступна для level 2 и 3) -->
    <!--v-if-->
    <!-- Users (доступна для level 3) -->
    <!--v-if-->
  </nav>
  <p style="width: 100%; height: 1px;"></p>
  <div style="align-items: center;">
    <h1>Добро пожаловать!</h1>
    <!-- <h2 style="width: 80%; align-content: flex-start;">
      Наша компания предлагает решения в сфере управления и автоматизации как для вашего дома, так и для вашего бизнеса.
    </h2>
    <h2 style="width: 80%; align-content: flex-end;">
      Вы можете выбрать один из разделов и узнать больше о типовых решениях или заказать индивидуальную разработку.
    </h2> -->
    <p>Выберите раздел:</p>
    <div class="introduce-place"><button class="introduce-button color_dark dash"><span class="button-text">Dashboard</span></button><button class="introduce-button color-light home"><span class="button-text">Smart Home</span></button><button class="introduce-button color-light manuft"><span class="button-text">Manufact Automation</span></button></div>
  </div>
</div>"
`;
